dropwizard-servers/src/main/java/com/palantir/remoting1/servers/DropwizardTracingFilters.java

```diff
@@ -54,25 +54,26 @@ private DropwizardTracingFilters() {}
      * <p>
      * TODO(rfink) Is there a more stable way to retrieve IP/Port information?
      */
-    static void registerTracers(Environment environment, Configuration config, String tracerName) {
-        ServerTracer serverTracer = getServerTracer(extractIp(config), extractPort(config), tracerName);
+    static Brave registerTracers(Environment environment, Configuration config, String tracerName) {
```

Comment:
I think we can add a simple Tracer interface with implementation backed by Brave for now. I should have Tritium ready shortly so that we can inject the tracer in there and wrap services as needed.